qf_lib/plotting/charts/waterfall_chart.py

```diff
@@ -0,0 +1,57 @@
+#     Copyright 2016-present CERN – European Organization for Nuclear Research
+#
+#     Licensed under the Apache License, Version 2.0 (the "License");
+#     you may not use this file except in compliance with the License.
+#     You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+from typing import Tuple, Optional, List
+
+import numpy as np
+
+from qf_lib.containers.series.qf_series import QFSeries
+from qf_lib.plotting.charts.chart import Chart
+
+
+class WaterfallChart(Chart):
+    def __init__(self, data: QFSeries, title: Optional[str] = None, total: Optional[List] = None):
+        super().__init__()
+        self.cumulative_sum = None
+        self.data = data
+        self.assert_is_qfseries(data)
+        self.title = title
+        self.total = total
+        self.color_green = '#3CB371'
+        self.color_red = '#FA8072'
+        self.color_blue = '#659FF1'
```

Comment:
Let's use for blue #4472C4, for red - #ED7D31 (orange) and instead of green - this grey  #A6A6A6 (so that it s exactly the same as the template we got)